OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -27,7 +27,7 @@ IntrumentedFunctionTypeFromOrbitType(FunctionInfo::OrbitType orbit_type) {
 
 ErrorMessageOr<void> CaptureClient::StartCapture(
     ThreadPool* thread_pool, int32_t pid,
-    const std::map<uint64_t, FunctionInfo*>& selected_functions) {
+    const absl::flat_hash_map<uint64_t, FunctionInfo>& selected_functions) {
```

Comment:
maybe pass by value here.. we are going to make a copy of this anyways.

Reply:
Done, now also move this into the lambda below.